### DISCUSSION TITLE
[packagers] choco bucket does not trigger workflow

### DISCUSSION THREAD

**Discussion by crschnick** [jreleaser/jreleaser#1325] [Q&A]
I have experimented with the chocolatey bucket feature and it does everything as expected, however for some reason no GitHub workflow is triggered when a push is done. The workflow files seem to be alright though.

**Comment by aalmiray** [comment]
FWIW this is the workflow used by JReleaser itself which has been working since we set it up more than a year ago https://github.com/jreleaser/chocolatey-bucket/blob/main/.github/workflows/push.yml

**Comment by crschnick** [comment]
Oh, it's probably because I use the `branchPush` property as the reference contains that. I will try just pushing to master

**Comment by crschnick** [comment]
It triggers now when the `branchPush` property is removed.

**Comment by crschnick** [comment]
So I would recommend to ether delete the property or not include it in the default configuration example in the documentation.

**Comment by aalmiray** [comment]
The property is generic to all taps/buckets/repositories. If a branch other than the default one is used (that is `branchPush` is defined) then the workflow should be updated as well.

**Reply by crschnick** [reply]
Are workflows that don't exist on master run when they are pushed to a separate branch by default?
